cloudflare/resource_cloudflare_page_rule.go

```diff
@@ -866,18 +866,23 @@ func transformToCloudflarePageRuleAction(id string, value interface{}, d *schema
 			for sectionID, sectionValue := range cacheKeyActionSchema[0].(map[string]interface{}) {
 				sectionOutput := map[string]interface{}{}
 
-				for fieldID, fieldValue := range sectionValue.([]interface{})[0].(map[string]interface{}) {
-					sectionOutput[fieldID] = fieldValue
+				if sectionValue.([]interface{})[0] != nil {
```

Comment:
just checking `""` and `0` aren't possible here either?

Reply:
I think they shouldn't.